README.md

```
@@ -130,21 +139,25 @@ BANTZ_GEMINI_MODEL=gemini-2.0-flash
+
+# Start HUD overlay (separate terminal)
+cd bantz-overlay && npm install && npm start
 ```
 
 <details>
-<summary>Google OAuth setup (for Calendar & Gmail)</summary>
+<summary>Google OAuth setup (Calendar, Gmail, Classroom)</summary>
 
 1. Place your Google Cloud OAuth client secret at `~/.config/bantz/google/client_secret.json`
 2. Authenticate:
    ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)